packages/platform/example/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: plaform_example
+description: Demonstrates how to use the plaform plugin.
+publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+version: 1.0.0+1
+
+environment:
+  sdk: '>=3.0.0 <4.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+  platform:
+    path: ../
+
+dev_dependencies:
+  flutter_lints: ^2.0.0
+  flutter_test:
+    sdk: flutter
+
+flutter:
+  uses-material-design: true
```

Comment:
Please add the missing newline, to avoid churn when people edit this file in an editor that adds it automatically.